fenix/app/src/main/res/values/strings.xml

```diff
@@ -1159,8 +1159,10 @@
     <string name="share_button_content_description">Share</string>
     <!-- Text for the Save to PDF feature in the share menu -->
     <string name="share_save_to_pdf">Save as PDF</string>
-    <!-- Text for error message when generating a PDF file Text for error message when generating a PDF file. -->
+    <!-- Text for snackbar error message when generating a PDF file. -->
     <string name="unable_to_save_to_pdf_error">Unable to generate PDF</string>
+    <!-- Text for error snackbar dismiss button when generating a PDF file. -->
+    <string name="unable_to_save_to_pdf_error_dismiss_button">Dismiss</string>
```

Comment:
```suggestion
    <string name="standard_snackbar_error_dismiss">Dismiss</string>
```
Maybe something like this phrasing, if we move to `ShowStandardSnackbarErrorAction`? I think the dismiss string would be universal and we just would need to pass an error message.

Reply:
Done

Reply:
Thanks!